etc/di.xml

```diff
@@ -28,4 +28,16 @@
             </argument>
         </arguments>
     </type>
+    <virtualType name="customStagedProductAttributeSubscription" type="Aligent\PrerenderIo\Model\Mview\View\Attribute\Subscription">
+        <arguments>
+            <argument name="entityInterface" xsi:type="string">Magento\Catalog\Api\Data\ProductInterface</argument>
+        </arguments>
+    </virtualType>
+    <type name="Magento\CatalogStaging\Model\Mview\View\SubscriptionFactory">
```

Comment:
Is it possible to use the `subscription_model` attribute within `mview.xml` instead of relying on the `Magento_CatalogStaging` module? Don't want to include an enterprise-only dependency if we can avoid it.